addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -362,15 +362,13 @@ function factory(dependencies) {
             );
             const inboxMailbox = this.env.messaging.inbox;
             inboxMailbox.update({ counter: inboxMailbox.counter + 1 });
-            for (const thread of message.allThreads) {
+            for (const thread of message.threads) {
                 if (
                     thread.channel_type === 'channel' &&
```

Comment:
Not sure but shouldn't this condition apply for non channel threads? If there is a needaction in a document thread?

Reply:
`message_needaction_counter` is unused for non-channels, so we could increment but this value is misleading because we are not aware of actual counter before increment/decrement of non-channels.

Reply:
There was a recent task in master to remove its display from kanban views but it was actually used before at least there.